protocol/consistencychecks.go

```diff
@@ -26,6 +27,7 @@ import (
 type ConsistencyChecks struct {
 	// SavedSTR stores the latest verified signed tree root.
 	SavedSTR *m.SignedTreeRoot
+	Bindings map[string][]byte
```

Comment:
I think we eventually need to have this map in `ConsistencyChecks`, and it makes sense as @masomel has pointed out. For now, there is no APIs to deal with this (just being used for storing verified bindings), but when we address the persistent storage from the client side, we may need to do something with this then. Maybe just unexport this for now?